kit/transport/http/middleware.go

```diff
@@ -53,9 +53,13 @@ func Metrics(name string, reqMetric *prometheus.CounterVec, durMetric *prometheu
 
 func SkipOptions(next http.Handler) http.Handler {
 	fn := func(w http.ResponseWriter, r *http.Request) {
-		if r.Method == "OPTIONS" {
+		// Preflight CORS requests from the browser will send an options request,
+		// so we need to make sure we satisfy them
+		if origin := r.Header.Get("Origin"); origin == "" {
+			w.WriteHeader(http.StatusMethodNotAllowed)
```

Comment:
love it 👍

Reply:
@rawkode we're not checking the Options method here 😳 just realized 🤣 

here's teh if block:

```go
if origin := r.Header.Get("Origin"); origin == "" && r.Method == http.MethodOptions {
```

Reply:
D'oh!

Reply:
Doesn't explain why my tests pass locally though 😯

Reply:
it failed for me locally before the fix. Maybe I gave you the wrong test cmd 😬